pgrx-pg-config/src/lib.rs

```diff
@@ -330,32 +334,55 @@ impl PgConfig {
 
     pub fn postmaster_path(&self) -> eyre::Result<PathBuf> {
         let mut path = self.bin_dir()?;
+        #[cfg(not(target_os = "windows"))]
         path.push("postgres");
-
+        #[cfg(target_os = "windows")]
+        path.push("postgres.exe");
         Ok(path)
     }
 
     pub fn initdb_path(&self) -> eyre::Result<PathBuf> {
         let mut path = self.bin_dir()?;
+        #[cfg(not(target_os = "windows"))]
         path.push("initdb");
+        #[cfg(target_os = "windows")]
+        path.push("initdb.exe");
         Ok(path)
     }
 
     pub fn createdb_path(&self) -> eyre::Result<PathBuf> {
         let mut path = self.bin_dir()?;
+        #[cfg(not(target_os = "windows"))]
         path.push("createdb");
+        #[cfg(target_os = "windows")]
+        path.push("createdb.exe");
         Ok(path)
     }
 
     pub fn dropdb_path(&self) -> eyre::Result<PathBuf> {
         let mut path = self.bin_dir()?;
+        #[cfg(not(target_os = "windows"))]
         path.push("dropdb");
+        #[cfg(target_os = "windows")]
+        path.push("dropdb.exe");
```

Comment:
Instead of this, can we get `format!("dropdb{EXE_SUFFIX}")` and so on?
```suggestion
        #[cfg(not(target_os = "windows"))]
        path.push("dropdb");
        #[cfg(target_os = "windows")]
        path.push("dropdb.exe");
```